src/BoomStick.h

```diff
@@ -1,9 +1,9 @@
 #pragma once
 #include <string>
 #include <map>
-#include "global.h"
-#include "boost/uuid/uuid.hpp"
-#include "boost/uuid/uuid_generators.hpp"
+#include <NMGlobal.h>
```

Comment:
We should break the link between our builds (NMTools or Research) and QueueNado. We should just replicate the bit of global.h file needed by QueueNado in a QueueNadoGlobal.h file.